packages/async-rewriter/src/async-writer-babel.spec.ts

```diff
@@ -347,7 +419,7 @@ class Test {
           it('decorates MemberExpression', (done) => {
             traverse(ast, {
               MemberExpression(path) {
-                expect(path.node['shellType']).to.deep.equal(signatures.Collection.attributes.insertOne);
+                expect(path.node['shellType']).to.deep.equal(signatures.Cursor.attributes.hasNext);
```

Comment:
These are only being changed because the type of `c` has changed, right?

Reply:
Yes, I just changed the test because `Collection` was being used as a test for a type that didn't have a proxy in it. Now it does, so I changed it to `Cursor`